src/requirements.txt

```diff
@@ -1 +1,2 @@
 PyYAML>=3.10,<4.0
+docker==2.2.1
```

Comment:
any chance 2.1.0 would be adequate? That would save (re)building it on fedora 26.

It looks like the deps are more or less in Fedora 25 and EL7 so it might not be hard to rebuilt the library for them. Mostly concern is if it causes conflicts with other installed packages on those releases require docker-py instead. python-docker-pycreds needs to be rebuilt for both python2 and 3 on EL7 as well and the same potential issue comes up.